src/main/resources/version.txt

```diff
@@ -0,0 +1 @@
+1.1.0-SNAPSHOT
```

Comment:
Why is this getting checked in? Wasn't this being generated by the build? And don't your changes make this file obsolete?

Reply:
Yes this shouldn't be there. It must have accidentally go thrown in there when switching between branches.